### ISSUE TITLE
The blur radius is too small when hiding cards in mobile mode

### ISSUE THREAD

**Issue by kimsey0** [microsoft/vsts-extension-retrospectives#355]
When using the function to blur cards in the collect phase in mobile mode (for example on a portrait monitor), the blur distance is too small to effectively hide the text
![Screenshot showing ineffectively blurred cards on a portrait monitor](https://user-images.githubusercontent.com/984760/170701555-5223f0c2-8254-4247-92be-7c3db09e7816.png)

**Comment by kimsey0** [comment]
Actually, looks like this may have been fixed by https://github.com/microsoft/vsts-extension-retrospectives/commit/071c37835d5eea63abf85bea6a184019b3cdb6d8, but just hasn't reached us yet.

**Comment by mindlessroman** [comment]
Yes! A better blur is a coming soon! I'm optimistic we can get a release in the next week sorted out :D

**Comment by mindlessroman** [comment]
I believe this was finally released @kimsey0!

**Comment by kimsey0** [comment]
Thanks. I had a look in our Azure DevOps tenant, and the blur radius still seems to be hard-coded to 5 pixels. However, the page will no longer switch to the mobile layout, no matter how I resize the window, nor if I view the page on a phone. Is this a new bug which I should report or was that the intended way to fix the blur issue?

**Comment by mindlessroman** [comment]
Dang, that may be a new bug. I'd say a new issue, if that's alright.
